content/docs/getting_started/cleanup.md

```diff
@@ -1,29 +1,34 @@
 ---
-title: "Cleanup"
+title: "Uninstall OSM"
 description: "Uninstall OSM and the bookstore applications"
 type: docs
 weight: 6
 ---
 
-# Cleanup
+# Uninstall OSM
 
-To cleanup all resources created for the demo, the OSM control plane, SMI resources, and the sample applications need to be deleted.
+The articles in the getting start section outline installing OSM and installing sample applications. To uninstall all of these resources, delete the sample application and related SMI resources and uninstall the OSM control plan and cluster-wide OSM resources.
```

Comment:
"The articles in the getting start section outline installing OSM and installing sample applications."
to
"The articles in the getting-started section outline installing OSM and sample applications."

Reply:
nice catch! fixed